content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/115-connection-pool.mdx

```diff
@@ -57,6 +57,115 @@ datasource db {
 }
 ```
 
+#### Viewing the connection pool size
+
+The number of connections Prisma Client uses can be viewed using [logging](/concepts/components/prisma-client/working-with-prismaclient/logging) and [metrics](/concepts/components/prisma-client/metrics).
+
+Using the `info` [logging level](/reference/api-reference/prisma-client-reference#log-levels)<span class="api"></span>, you can log the number of connections in a connection pool that are opened when Prisma Client is instantiated.
+
+For example, consider the following Prisma Client instance and invocation:
+
+<CodeWithResult expanded="{true}">
+<cmd>
+
+```ts
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient({
+  log: ['info'],
+})
+
+async function main() {
+  await prisma.user.findMany()
+}
+
+main()
+```
+
+</cmd>
+<cmdResult>
+
+```text no-copy
+prisma:info Starting a postgresql pool with 21 connections.
+```
+
+</cmdResult>
+</CodeWithResult>
+
+When the `PrismaClient` class was instantiated, the logging notified `stdout` that a connection pool with 21 connections was started.
+
+If you need even more insights into the size of your connection pool and the amount of in-use and idle connection, you can use metrics.
```

Comment:
Note that metrics is still a preview feature, so that should be mentioned here.

Reply:
how about this?

```suggestion
If you need even more insights into the size of your connection pool and the amount of in-use and idle connection, you can use the [metrics](/concepts/components/prisma-client/metrics) feature (which is currently in Preview).
```